gammapy/catalog/fermi.py

```diff
@@ -556,22 +569,177 @@ class SourceCatalogObject3FHL(SourceCatalogObject):
-        ss += 'Detection significance : {}\n'.format(d['Signif_Avg'])
+        ss += '{:<20s} : {:.4f}\n'.format('Semimajor (95%)', d['Conf_95_SemiMajor'])
+        ss += '{:<20s} : {:.4f}\n'.format('Semiminor (95%)', d['Conf_95_SemiMinor'])
+        ss += '{:<20s} : {:.2f}\n'.format('Position angle (95%)', d['Conf_95_PosAng'])
+        ss += '{:<20s} : {:.0f}\n'.format('ROI number', d['ROI_num'])
+
+        return ss;
+
+    def _info_spectral_fit(self):
+        """Print model data."""
+        d = self.data
+        ss = '\n*** Model info ***\n\n'
+
+        fmt = '{:<32s} : {:.3f}\n'
+        args = ('Significance (10 GeV - 2 TeV)', d['Signif_Avg'])
+        ss += fmt.format(*args)
+
+        ss += '{:<32s} : {:.1f}\n\n'.format('Npred', d['Npred'])
+
+
+        ss += '{:<32s} : {}\n'.format('Spectrum type', d['SpectrumType'])
+
+        ss += '{:<32s} : {:.1f}\n'.format('Significance curvature', d['Signif_Curve'])
+
+        fmt = '{:<32s} : {:.3f} +- {:.3f}\n'
+        args = ('Spectral index', d['Spectral_Index'], d['Unc_Spectral_Index'])
+        ss += fmt.format(*args)
+
+        spec_type = d['SpectrumType'].strip()
+        if spec_type == 'LogParabola':
+            # ss += '{:<32s} : {:.3f} +- {:.3f}\n'.format('alpha', d['Spectral_Index'], d['Unc_Spectral_Index'])
+            ss += '{:<32s} : {:.3f} +- {:.3f}\n'.format('beta', d['beta'], d['Unc_beta'])
+
+        ss += '{:<32s} : {:.0f} {}\n'.format('Pivot energy', d['Pivot_Energy'].value, d['Pivot_Energy'].unit)
+
+        fmt = '{:<32s} : {:.3f} +- {:.3f}\n'
+        args = ('Power Law index', d['PowerLaw_Index'], d['Unc_PowerLaw_Index'], d['Unc_PowerLaw_Index'])
+        ss += fmt.format(*args)
+
+        unit = 'cm-2 GeV-1 s-1'
+        fmt = '{:<32s} : {:.3} +- {:.3} {}\n'
+        args = ('Flux Density at pivot energy', d['Flux_Density'].value, d['Unc_Flux_Density'].value, unit)
+        ss += fmt.format(*args)
+
+        unit = 'cm-2 s-1'
+        fmt = '{:<32s} : {:.3} +- {:.3} {}\n'
+        args = ('Integral flux (10 GeV - 1 TeV)', d['Flux'].value, d['Unc_Flux'].value, unit)
+        ss += fmt.format(*args)
+
+        unit = 'erg cm-2 s-1'
+        fmt = '{:<32s} : {:.3} +- {:.3} {}\n'
+        args = ('Energy flux (10 GeV - TeV)', d['Energy_Flux'].value, d['Unc_Energy_Flux'].value, unit)
+        ss += fmt.format(*args)
+
+        return ss
+
+    def _info_spectral_points(self):
+        """Print spectral points."""
+        ss = '\n*** Spectral points ***\n\n'
+
+        t = self._flux_points_table_formatted
+
+        ss += '\n'.join(t.pformat(max_width=-1))
+        return ss + '\n'
+
+    def _info_other(self):
+        """Print other info."""
+        d = self.data
+        ss = '\n*** Other info ***\n\n'
+
+        ss += '{:<16s} : {:.3f} {}\n'.format('HEP Energy', d['HEP_Energy'].value, d['HEP_Energy'].unit)
+        ss += '{:<16s} : {:.3f}\n'.format('HEP Probability', d['HEP_Prob'])
+
+        bayes = d['Variability_BayesBlocks']
+        str = bayes
```

Comment:
This code can be improved a bit:
- First of all `str` is a Python built-in keyword. Overwriting Python built-ins with variable names is considered bad practice, so you should choose some other variable name here like `msg` or `message` or anything else you like.
- Secondly, you're creating two local variables "str" and "bayes", but you really only need one.